src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -89,6 +89,7 @@ class UnusedTagsPass implements CompilerPassInterface
         'validator.auto_mapper',
         'validator.constraint_validator',
         'validator.initializer',
+        'web_pusher.transport_factory',
```

Comment:
Is it a tag?

Reply:
I've added it in same way as `chatter.transport_factory`